app/assets/stylesheets/analytics/analytics.scss

```diff
@@ -0,0 +1,149 @@
+#analytics-dashboard {
+  width: 1170px;
+  .totals {
+    position: relative;
+
+    dl {
+      width: 370px;
+    }
+
+    dt {
+      font-size: 95px;
+    }
+
+    dd {
+      font-size: 30px;
+      border-top: 1px solid #ddd;
+    }
+  }
+
+  .analytics-chart {
+    margin-top: 26px;
+  }
+
+  .chart-title {
+    margin-top: 0px;
+  }
+
+  .mini {
+    top:      46px;
+    position: absolute;
+    margin:   0;
+    padding:  0;
+  }
+
+  .axis {
+    font: 10px sans-serif;
+  }
+
+  .axis path,
+  .axis line {
+    fill: none;
+    shape-rendering: crispEdges;
+  }
+
+  .x.axis path {
+    display: none;
+  }
+
+  .hide-tick {
+    display: none;
+  }
+}
+.odometer .odometer-inside {
+  border-top: 1px solid #ddd;
+  width: 370px;
+}
+
+.odometer.odometer-auto-theme, .odometer.odometer-theme-default {
+  display: inline-block;
+  vertical-align: middle;
+  *vertical-align: auto;
+  *zoom: 1;
+  *display: inline;
+  position: relative;
+}
+.odometer.odometer-auto-theme .odometer-digit, .odometer.odometer-theme-default .odometer-digit {
+  display: inline-block;
+  *vertical-align: auto;
+  *zoom: 1;
+  *display: inline;
```

Comment:
are you going IE hacks or are you copy-pasting the library's css into our repo? doesn't rails-assets just let us import odometer's css directly with sprockets?